boot/zephyr/Kconfig

```diff
@@ -393,6 +405,13 @@ config RECOVERY_UART_DEV_NAME
 
 endif # MCUBOOT_SERIAL
 
+config UPDATEABLE_IMAGE_NUMBER
```

Comment:
Please include copy of https://github.com/JuulLabs-OSS/mcuboot/commit/16b6d15a6b0dc8fe723b47e8fcf5f742f4f002e6
This option is duplicated in current patch version.